go.mod

```diff
@@ -3,6 +3,7 @@ module knative.dev/net-contour
 go 1.14
 
 require (
+	github.com/dprotaso/dekupe v0.0.0-20201119194102-ec62861e5660
```

Comment:
Can this lives in our Knative automation repo  or test-infra?

cc @chaodaiG @n3wscott @mattmoor

Reply:
Sure. What does the tool do?

Reply:
it dedupes k8s objects in yaml